README.md

```diff
@@ -133,13 +122,18 @@ To use `glutin`, for example, your `Cargo.toml` must be extended with the follow
 
 Alternatively, an excellent introduction into gfx and its related crates can be found [here](https://wiki.alopex.li/LearningGfx).
 
+## Running the Examples
+
+The [examples directory](./examples) contains all the examples for GFX, as well as the
+[accompanying documentation](./examples/README.md) for understanding and running those examples.
+
+
 ## Structure and current versions
 `gfx` consist of several crates. You can find all of them in this repository.
 
 | Core functionality: | Graphic backends: | Window backends: |
 | :---: | :---: | :---: |
 | [![gfx on crates.io](http://img.shields.io/crates/v/gfx.svg?label=gfx)](http://crates.io/crates/gfx) | [![gfx_device_gl on crates.io](http://img.shields.io/crates/v/gfx_device_gl.svg?label=gfx_device_gl)](http://crates.io/crates/gfx_device_gl) | [![gfx_window_sdl on crates.io](http://img.shields.io/crates/v/gfx_window_sdl.svg?label=gfx_window_sdl)](http://crates.io/crates/gfx_window_sdl) |
-| [![gfx_app on crates.io](http://img.shields.io/crates/v/gfx_app.svg?label=gfx_app)](http://crates.io/crates/gfx_app) | [![gfx_device_dx11 on crates.io](http://img.shields.io/crates/v/gfx_device_dx11.svg?label=gfx_device_dx11)](http://crates.io/crates/gfx_device_dx11) | [![gfx_window_dxgi on crates.io](http://img.shields.io/crates/v/gfx_window_dxgi.svg?label=gfx_window_dxgi)](http://crates.io/crates/gfx_window_dxgi) |
```

Comment:
It was indicated earlier in the discussion that it was desirable to kill of gfx_app all together.